### DISCUSSION TITLE
Win-KeX Doesn't start

### DISCUSSION THREAD

**Discussion by OTH315** [microsoft/WSL#9675] [General]
When i type kex it shows this massage:
Win-KeX server sessions:

X DISPLAY #     RFB PORT #      RFB UNIX PATH   PROCESS ID #    SERVER
1               5901                            707             Xtigervnc

You can use the Win-KeX client to connect to any of these displays.
this massege is normal but i keep waiting for an 1h and its still not connected. if you know how to solve this pls tell me

**Comment by josemtella** [comment]
https://jmtella.com/foro/forum/ejercicios-y-ejemplos/49620-tip-kali-kex-soluci%C3%B3n-a-los-fallos-con-las-ultimas-versiones-de-wsl
